From 16dd9e4a5532ad93d8af93a27ebe27c4c07fffd2 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Kai=20K=C3=B6hne?= <kai.koehne@qt.io>
Date: Tue, 14 Oct 2025 15:52:44 +0200
Subject: [PATCH 424/553] Doc: Fix CMake snippet on how to import Qt6*Private

Commit e120fb78c94124fac0f9d381875c9de0721108ba does change the
CMake API so that public Qt modules don't provide the
targets of their private counterparts anymore. Adapt the snippets
shown for CorePrivate, GuiPrivate accordingly.

Task-number: QTBUG-141152
Change-Id: I357f1e737e431c8e111dca2594b886e7567983cc
Reviewed-by: Joerg Bornemann <joerg.bornemann@qt.io>
(cherry picked from commit 32f42d1cd457569f76b6bd030e16e51d5d7ffc7a)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/corelib/doc/src/qtcore.qdoc | 3 +--
 src/gui/doc/src/qtgui.qdoc      | 3 +--
 2 files changed, 2 insertions(+), 4 deletions(-)

diff --git a/src/corelib/doc/src/qtcore.qdoc b/src/corelib/doc/src/qtcore.qdoc
index 38d73870af5..ec5fa564639 100644
--- a/src/corelib/doc/src/qtcore.qdoc
+++ b/src/corelib/doc/src/qtcore.qdoc
@@ -19,8 +19,7 @@
     \module QtCorePrivate
     \title Qt Core Private C++ Classes
     \qtvariable core-private
-    \qtcmakepackage Core
-    \qtcmaketargetitem CorePrivate
+    \qtcmakepackage CorePrivate
     \preliminary
     \brief Provides private core functionality.
 
diff --git a/src/gui/doc/src/qtgui.qdoc b/src/gui/doc/src/qtgui.qdoc
index a89872e70f6..f5d60699deb 100644
--- a/src/gui/doc/src/qtgui.qdoc
+++ b/src/gui/doc/src/qtgui.qdoc
@@ -17,8 +17,7 @@
 /*!
     \module QtGuiPrivate
     \title Qt GUI Private C++ Classes
-    \qtcmakepackage Gui
-    \qtcmaketargetitem GuiPrivate
+    \qtcmakepackage GuiPrivate
     \qtvariable gui-private
 
     \brief Provides access to private GUI functionality.
-- 
2.51.2

